query-engine/src/main.py

```
@@ -1,5 +1,5 @@
 from contextlib import asynccontextmanager
-from fastapi import FastAPI, HTTPException
+from fastapi import FastAPI, HTTPException, Request
```

Ruff (0.8.2)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)